.github/workflows/gradle.yml

```diff
@@ -247,18 +247,8 @@ jobs:
       #      - name: Get Docker Space
       #        run: docker run --rm busybox df -h
 
-      - name: Image Cleanup
```

Comment:
these should not be needed anymore with the slimmer build.

Reply:
good catch. should we also remove the script? or keep just in case?